extension/src/test/fixtures/expShow/base/customPlots.ts

```diff
@@ -34,88 +34,158 @@ export const customPlotsOrderFixture: CustomPlotsOrderValue[] = [
   }
 ]
 
-export const checkpointPlotsFixture: CustomCheckpointPlots = {
-  'summary.json:loss': {
-    id: 'custom-summary.json:loss-epoch',
-    metric: 'summary.json:loss',
-    param: CHECKPOINTS_PARAM,
-    type: CustomPlotType.CHECKPOINT,
-    values: [
-      { group: 'exp-83425', iteration: 1, y: 1.9896177053451538 },
-      { group: 'exp-83425', iteration: 2, y: 1.9329891204833984 },
-      { group: 'exp-83425', iteration: 3, y: 1.8798457384109497 },
-      { group: 'exp-83425', iteration: 4, y: 1.8261293172836304 },
-      { group: 'exp-83425', iteration: 5, y: 1.775016188621521 },
-      { group: 'exp-83425', iteration: 6, y: 1.775016188621521 },
-      { group: 'test-branch', iteration: 1, y: 1.9882521629333496 },
-      { group: 'test-branch', iteration: 2, y: 1.9293040037155151 },
-      { group: 'test-branch', iteration: 3, y: 1.9293040037155151 },
-      { group: 'exp-e7a67', iteration: 1, y: 2.020392894744873 },
-      { group: 'exp-e7a67', iteration: 2, y: 2.0205044746398926 },
-      { group: 'exp-e7a67', iteration: 3, y: 2.0205044746398926 }
-    ]
-  },
-  'summary.json:accuracy': {
-    id: 'custom-summary.json:accuracy-epoch',
-    metric: 'summary.json:accuracy',
-    param: CHECKPOINTS_PARAM,
-    type: CustomPlotType.CHECKPOINT,
-    values: [
-      { group: 'exp-83425', iteration: 1, y: 0.40904998779296875 },
-      { group: 'exp-83425', iteration: 2, y: 0.46094998717308044 },
-      { group: 'exp-83425', iteration: 3, y: 0.5113166570663452 },
-      { group: 'exp-83425', iteration: 4, y: 0.557449996471405 },
-      { group: 'exp-83425', iteration: 5, y: 0.5926499962806702 },
-      { group: 'exp-83425', iteration: 6, y: 0.5926499962806702 },
-      { group: 'test-branch', iteration: 1, y: 0.4083833396434784 },
-      { group: 'test-branch', iteration: 2, y: 0.4668000042438507 },
-      { group: 'test-branch', iteration: 3, y: 0.4668000042438507 },
-      { group: 'exp-e7a67', iteration: 1, y: 0.3723166584968567 },
-      { group: 'exp-e7a67', iteration: 2, y: 0.3724166750907898 },
-      { group: 'exp-e7a67', iteration: 3, y: 0.3724166750907898 }
+export const experimentsWithCheckpoints: ExperimentWithCheckpoints[] = [
```

Comment:
Do we have data similar to this in a different spot in the codebase? I couldn't find any and created one based off our current `CustomPlotFixture`.

Reply:
nothing significant enough